VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -59,6 +59,7 @@
     private List<DiscussionComment> discussionResponses = new ArrayList<>();
 
     private boolean progressVisible = false;
+    private final long initialTimeStampMs = System.currentTimeMillis();
```

Comment:
Maybe add a comment before this stating the purpose of this, e.g.:

````java
// Record the current time at initialization to keep the elapsed time durations display stable.
````

This applies to the other adapter as well.